src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -48,7 +48,9 @@ public void setBenchmarkName(String benchmarkName) {
   private String db_username;
   private String db_password;
   private String db_driver;
-  private double scaleFactor = 1.0;
+  private int scaleFactor = -1;
```

Comment:
Comment that this is the number of warehouses (or better yet rename the variable)

Reply:
Done.